add k2 content strategy draft (#565)
kyma-project/community: guidelines/templates/resources/task.md

---
title: {Document title}
---

<!-- Use this template to write "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure. 

You can use this template for any step-by-step instruction, no matter whether it's a task during the getting started guide, a tutorial for software developers, or an operational guide.

As document file name, follow the pattern {COMPONENT_ABBRV}-{NUMBER_PER_COMPONENT}-{FILE_NAME}.md}.

> Select a title that describes the task that's accomplished, not the documented software feature - for example, "Define ressource consumption", not "Select a profile". You can use gerund form "Selecting...", imperative "Select...", or "How to select...".

Reviewer: ```suggestion
Select a title that describes the task that's accomplished, not the documented software feature. For example, use "Define resource consumption", not "Select a profile". You can use the gerund form "Selecting...", imperative "Select...", or "How to select...".
```

With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
It's good practice to have 5-9 steps; anything longer can probably be split.
-->

## Context

<!-- Briefly provide background information for the task so that the users understand the purpose of the task and what they will gain by completing the task correctly. This section should be brief and does not replace or recreate a concept topic on the same subject, although the context section might include some conceptual information.
-->

## Prerequisites

> <!-- Describes information that the user needs to know or do before starting the immediate task. 
> If it's more than one prerequisite, use an unordered list.
> For example, specify the authorizations the user must have and which software (and versions) must be installed already.
>  -->

Reviewer: ```suggestion
<!-- Describes information that the user needs to know or things they need to do or have before starting the immediate task. 
If it's more than one prerequisite, use an unordered list.
For example, specify the authorizations the user must have and what software (and versions) must be installed already.
 -->
```

- 
- 
- 

## Procedure

<!-- Provide a series of steps needed to perform the task. 
Use a numbered list with one number for each action that the users must take. 

It's good practice to describe the result of the procedure so that the users can see they accomplished the task successfully. 

> Sometimes it's also very helpful to describe the result of a specific step (don't use a number for step results, just a new line below the step).

Reviewer: ```suggestion
Sometimes it's also very helpful to describe the result of a specific step (don't use a number for step results, just a new line below the step). Remember about appropriate indentation for this line.
```

Author: This!! Thank you

> If the task at hand typically is followed by another one, you can add a link to that other document as "Next Steps".

Reviewer: ```suggestion
If the task at hand is typically followed by another one, you can add a link to that other document as "Next Steps".
```

-->

1. 
2. 
3. 

> Result: 

Reviewer 2: Should `Result` and `Next Steps` be separate sections as well? Are `Next Steps` obligatory or not? I can imagine there are no next steps for some tutorials.

Author: I consider both optional, but `Result` very recommended. 
I agree, `Next Steps` are only useful in some cases, but in such cases it makes sense to encourage adding the link. 

I'm adding this info
